lib/http.js

```diff
@@ -272,38 +273,31 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
     routes.automatedIssue,
     // TODO: validate body
     asyncHandler(async (req, res) => {
-
-      // TODO: inspect/validate API token
-      // token will need to align with issuer.id
-
       const {
-        issuer,
-        credentials,
-      } = req.body;
-
-      // get instance for issuer
-      const {instance} = await vcIssuer.instances.get(issuer);
-      const {controller: accountId} = instance;
+        body: {credential},
+        headers: {authorization},
+      } = req;
+
+      const tokenStartIndex = 7;
+      if(!(authorization && typeof authorization === 'string' &&
+        authorization.startsWith('Bearer ') &&
+        authorization.length > tokenStartIndex)) {
+        throw new BedrockError('Permission denied.', 'NotAllowedError', {
+          httpStatusCode: 400,
+          public: true,
+        });
+      }
 
-      // get associated configuration
+      const token = authorization.substr(tokenStartIndex);
+      const profileAgentRecord = await profileAgents.getByToken(
+        {token, includeSecrets: true});
 
-      // TODO: a configuration will contain a schema that the incoming
-      // credential will be validated against
-      // const configuration = await vcIssuer.configurations.get({instance});
```

Comment:
I'm concerned about losing these TODOs ... are we tracking elsewhere?

Reply:
I guess this one isn't that big of a deal and the others are handled or things have changed. Dismissing.